2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 2 Line 20_
```markdown
* `list`: a mutable ordered list of data values
* `tuple`: an immutable, unordered list of data values
* `set`: an unordered mutable collection of unique data values
* `dict`: a hash map which permits lookup on the basis of keys and values

We will explore each of these data types and containers in this module. 

### Type checking
```
To avoid confusion, maybe call this "Type inspection" or just "The type function"?

Technically type checking is like the automatic enforcement/testing of code with type annotations. I have a section in module 3 on that (or rather why we're not teaching it!)